.github/workflows/sonar.yaml

```diff
@@ -3,6 +3,7 @@ on:
   push:
     branches:
       - main
+      - 5.x # 5.x supports JDK 17
```

Comment:
We use to only run sonar on main (active dev branch). Do we need to run on 5.x?

Reply:
Not really. It makes sense to only run on main. I'll revert this line.